### PR TITLE
Add footer api contract

### PR DESCRIPTION
**Pull request by goelsonali**
Footer page information

Related to #9

screnshot 

Screenshot 2024-06-06 at 12 52 58

### CODE REVIEW

File: api_module_about.yml [hunk 1]
```diff
@@ -90,6 +90,7 @@ paths:
         default:
           $ref: 'schemas/responses.yml#/components/responses/Forbidden'
 
+
```
Comment (dricazenck) [inline api_module_about.yml:93]: Please update your branch, this file does not exist. (I renamed)

File: parameters/page.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: page
```
Comment (dricazenck) [inline parameters/page.yml:1]: I guess this change is not related to footer, right?

File: parameters/pageSize.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: pageSize
+in: query
+description: Number of items per page
+required: false
+schema:
```
Comment (dricazenck) [inline parameters/pageSize.yml:5]: Same here

File: parameters/page.yml (hunk 2 above)
Comment (goelsonali) [reply parameters/page.yml:1]: No but it was improvement to the parameter section

File: parameters/pageSize.yml (hunk 3 above)
Comment (goelsonali) [reply parameters/pageSize.yml:5]: No but it was improvement to the parameter section
